CMakeLists.txt

```diff
@@ -197,27 +238,6 @@ set(libc++_include_dir "${libwebrtc_source_dir}/src/buildtools/third_party/libc+
 add_library(libc++abi OBJECT IMPORTED)
 add_dependencies(libc++abi libwebrtc)
 
-set(libc++abi_objects
-  abort_message.o
-  cxa_aux_runtime.o
-  cxa_default_handlers.o
-  cxa_demangle.o
-  cxa_exception.o
-  cxa_exception_storage.o
-  cxa_guard.o
-  cxa_handlers.o
-  cxa_personality.o
-  cxa_unexpected.o
-  cxa_vector.o
-  cxa_virtual.o
-  fallback_malloc.o
-  private_typeinfo.o
-  stdlib_exception.o
-  stdlib_stdexcept.o
-  stdlib_typeinfo.o
-)
-list(TRANSFORM libc++abi_objects PREPEND ${libwebrtc_binary_dir}/obj/buildtools/third_party/libc++abi/libc++abi/)
-
```

Comment:
Nit: do you mind just moving the entire libc++ section to before the libwebrtc section, rather than splitting out a "libc++ object list" section? I see that `libwebrtc_source_dir` is needed for the remaining lines, but we could defiine that the same as you've done `libwebrtc_binary_dir`.

Reply:
I've moved almost all of the libc++ section to before the libwebrtc section. I've had to keep the lines that set `libc++_include_dir` and `libc++abi_include_dir` after the libwebrtc section to prevent the build from failing.